docs/production-deployment/cloud/metrics/index.mdx

```diff
@@ -27,8 +27,18 @@ tags:
   - temporal-cloud
 ---
 
-Beyond the [metrics](/references/sdk-metrics) provided by the Temporal SDKs, some key metrics exist only in Temporal Cloud.
-You can use your own observability tool to query an endpoint and review Namespace metrics.
+Temporal offers two distinct sources of metrics: [Cloud/Server Metrics](/production-deployment/cloud/metrics/reference) and [SDK Metrics](/references/sdk-metrics).
+Each source provides options for levels of granularity and filtering, monitoring-tool integrations, and configuration.
+Before implementing observability for Temporal Cloud, decide what you need to measure for your use case. There are two primary use cases for metrics:
```

Comment:
Consider "Temporal Cloud observability"

Second sentence, if you want to get rid of passive voice, "Use metrics to" (and adjust bullets).